scheduler/src/main/java/gov/nasa/jpl/aerie/scheduler/Problem.java

```diff
@@ -60,7 +59,15 @@ public Problem(MissionModel<?> mission, PlanningHorizon planningHorizon) {
         this.add(new ActivityType(taskType.getKey(), taskType.getValue()));
       }
     }
-    simulationFacade = new SimulationFacade(planningHorizon, missionModel);
+    this.simulationFacade = simulationFacade;
+  }
+
+  public Problem(PlanningHorizon planningHorizon){
+    this(null, planningHorizon, null);
```

Comment:
Just trying to understand - why do we want to allow defining a Problem with no mission model, and no simulation facade?

Reply:
For toy problems, for tests.

Reply:
I would like to find a way to make the interface consistent for "real" problems and "test" problems. Not sure off the top of my head how to do that.

Reply:
I made a ticket for this, no need to do this in this PR: AERIE-1748